scripts/postData.sh

```diff
@@ -32,4 +32,4 @@ OUTPUT_FILE="${ALLSKY_TMP}/${FILE}"
 ) > "${OUTPUT_FILE}"
 
 # PD == Post Data
-"${ALLSKY_SCRIPTS}/upload.sh" --silent "${OUTPUT_FILE}" "${IMG_DIR}" "${FILE}" "PD"
+"${ALLSKY_SCRIPTS}/upload.sh" --silent "${OUTPUT_FILE}" "${IMGDIR}" "${FILE}" "PD"
```

Comment:
Doesn't this go the wrong direction?  I thought we were moving to IMAGE_DIR?
Also, should we use VIDEO_DIR vs VIDEOS_DIR ?  The other items are singular vs plural. (well, startrails looks plural, but... lol )

Reply:
The eventual name in ftp-settings.sh will be IMAGE_DIR, but for backward compatibilities' sake, I'm still using the old IMGDIR name.  My concern is that people will download the updated scripts but not the updated ftp-settings.sh-repo or won't update their existing ftp-settings.sh file, so they'll be using the old IMGDIR name.

The name of the new MP4DIR should be VIDEOS_DIR (plural) since /videos is the name of the directory on the Pi.  Does some script have VIDEO_DIR?

Reply:
na, was just a style observation.  Hopefully, users are downloading the whole repo.  I agree with keeping a comparability variable in `ftp-settings.sh.repo`, but if we're updating the scripts, we should move to the new name ( imo ).